src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -76,8 +76,8 @@ def load_arguments(self, _):
         c.argument('min_count', type=int, validator=validate_nodes_count)
         c.argument('max_count', type=int, validator=validate_nodes_count)
         c.argument('api_server_authorized_ip_ranges', type=str, validator=validate_ip_ranges)
-        c.argument('enable_pod_security_policy', action='store_true', options_list=['--enable-pod-security-policy'])
-        c.argument('disable_pod_security_policy', action='store_true', options_list=['--disable-pod-security-policy'])
+        c.argument('enable_pod_security_policy', action='store_true')
+        c.argument('disable_pod_security_policy', action='store_true')
```

Comment:
It would likely make more sense to have one parameter `--enable-pod-security-policy` that is a three-state flag and can be turned off by using `--enable-pod-security-policy false`. Otherwise, what do you do if the user supplies both flags?

Reply:
If user specifies both flags, the CLI with [return an error](https://github.com/Azure/azure-cli-extensions/pull/604/files#diff-071fc9e293b8a43c36af8e578fe122e4R564). It is designed to keep aligned with other flags: `enable-rbac`&`disable-rbac`, `enable-cluster-autoscaler`&`disable-cluster-authscaler`, does it make sense?

Reply:
If the command already followed this anti-pattern, then it would make sense to stay consistent with itself.